downstream/modules/platform/con-aap-migration-prereqs.adoc

```diff
@@ -0,0 +1,16 @@
+[id="aap-migration-prereqs"]
+
+= Prerequisites
+To migrate {PlatformNameShort} deployment to {OperatorPlatform}, you must have the following:
+
+[role="_abstract"]
+
+* Secret key secret
+* Postgresql configuration
+* Role-Based Access Controls
```

Comment:
This might be too vague.  Maybe "Role-based Access Control for the namespace(s) on the new Openshift cluster"

Specifically, they need to be able to create k8s secrets and install an Operator from OperatorHub. That will probably become apparent to them as they read on though.

Reply:
Suggested changes complete.